testing/src/test/java/io/stargate/it/CQLTest.java

```diff
@@ -350,6 +364,52 @@ public void tupleTest()
         assertThat(tupleReturnValue.getInt(2)).isEqualTo(2);
     }
 
+    @Test
+    public void compressionTest()
+    {
+        Arrays.stream(new String[] {"lz4", "snappy"}).forEach(compression -> {
+            DriverConfigLoader loader =
+                DriverConfigLoader.programmaticBuilder()
+                    .withBoolean(DefaultDriverOption.METADATA_TOKEN_MAP_ENABLED, false)
+                    .withString(DefaultDriverOption.LOAD_BALANCING_POLICY_CLASS, DcInferringLoadBalancingPolicy.class.getName())
+                    .withDuration(DefaultDriverOption.REQUEST_TIMEOUT, Duration.ofSeconds(5))
```

Comment:
maybe extract this logic to a separate method? (since it is duplicated)

Reply:
good call, will do.

Reply:
done!